themes/bootstrap3/templates/Related/Bookplate.phtml

```diff
@@ -0,0 +1,26 @@
+<?php $bookplates = $this->related->getBookplateDetails(); ?>
+<div id="bookplates">
+  <?php foreach($bookplates as $bookplate): ?>
+    <?php $bpTitle = $bookplate['title'];?>
+    <?php $bpAltText = $bpTitle . ' bookplate';?>
```

Comment:
We need a translation for this alt text -- I would suggest adding something to en.ini like:

<pre>
bookplate_label = "%%title%% bookplate"
</pre>

And then changing this to something like:

<pre>
$bpAltText = $this->translate('bookplate_label', ['%%title%%' => $bpTitle]);
</pre>

This also raises the question of whether $bpTitle should also be run through $this->translate in case somebody wants to set up a multi-lingual bookplate title.

Reply:
I _really_ like the idea of running `$bpTitle` through `translate`. It adds a much greater degree of flexibility. I never would have thought of this but even even for people who might not want to create a multi-lingual bookplate title, I can imagine an edge case scenario where they might want to control all the titles through a language file. For example, if someone only had donor codes, like we have, and nothing else. It might not be an ideal practice but they could actually do everything with just 1 field if I'm not mistaken. Running `$bpTitle` through `translate` makes that possible.

Reply:
That's great! I wonder if it would actually make sense to use a separate text domain for bookplate titles, to avoid any risk of unexpected translation based on existing values in the language file. You could just prepend the title with "bookplate::" in the translate call, and then the translator would look for translations in `languages/bookplate/en.ini` instead of `languages/en.ini`. (And, of course, in the default case where no translations are defined, the behavior should remain exactly the same as ever).

Reply:
Yes, definitely. I didn't know that was an option but I think it makes sense. If someone were to actually manage their bookplate titles in this way, it would give them a nice separate place to keep everything.